legion/services/legion/services/edi/server.py

```diff
@@ -313,8 +313,11 @@ def generate_token(model_id, model_version, expiration_date=None):
     """
     Generate JWT token
 
-    :return: dict -- state of cluster
+    :return: dict -- JWT token and expiration information
     """
+    if not app.config['JWT_CONFIG']:
+        return {'token': '', 'exp': None}
```

Comment:
What do you think about showing some message in `legionctl` when the token is empty?
For example:
```bash
$ legionctl generate-token ...
JWT integration is disabled
```

Reply:
Agree, I'll do it

Reply:
done